contracts/Config.sol

```diff
@@ -33,4 +36,8 @@ contract Config is IConfig, OwnableUpgradeable {
     function getFeeReceiver() external view returns (address) {
         return feeReceiver;
     }
+
+    function getValidForDays() external view returns (uint256) {
```

Comment:
Actually no particular reason for that, it was just because I had created the interface of Config and used it in ScheduledPaymentModule before I created the implementation of Config, do you think it's better to use the autogenerated getter?

Reply:
No strong preference, this is clear and simple, maybe it uses a small amount less bytecode for the contract but I'm not sure of that